template/context.go

```diff
@@ -14,6 +14,12 @@ import (
 	"github.com/pkg/errors"
 )
 
+// Platform holds information about the underlying platform of the node
+type Platform struct {
```

Comment:
Forget I said that... realized this is going into a struct that will be accessable by templates, so no methods should be defined on the type.